test/parallel_api/dynamic_selection/sycl/test_internal_sycl_scheduler.pass.cpp

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Did not review tests either. Generic suggestion: tests should not rely on implementation details. I thought I saw those some time ago but if this is not the case anymore - that's good.